WalletWasabi.Fluent/ViewModels/Wallets/Send/LabelSelectionViewModel.cs

```diff
@@ -65,11 +68,16 @@ public Pocket[] GetUsedPockets()
 
 	public void Reset(Pocket[] pockets)
 	{
-		_privatePocket = pockets.FirstOrDefault(x => x.Labels == CoinPocketHelper.PrivateFundsText);
+		_allPockets = pockets;
+
+		if (pockets.FirstOrDefault(x => x.Labels == CoinPocketHelper.PrivateFundsText) is { } privatePocket)
+		{
+			_privatePocket = privatePocket;
+		}
 
-		AllPockets = pockets.Where(x => x != _privatePocket).ToArray();
+		NonPrivatePockets = pockets.Where(x => x != _privatePocket).ToArray();
 
-		var allLabels = SmartLabel.Merge(AllPockets.Select(x => x.Labels));
+		var allLabels = SmartLabel.Merge(NonPrivatePockets.Select(x => x.Labels));
```

Comment:
Yes, only the private funds are excluded because that is not an entity, can't be handled as an entity so they can't be shown on the view.
So `allLabels` are the labels that can be used on the view to interact with.